index.md

```diff
@@ -35,7 +35,7 @@ Graphical builds run much faster with some kind of graphics acceleration.  2GB o
 
 ## Downloads
 
-Cataclysm has official builds for Windows, Linux and OSX, and each platform comes in both terminal and graphical flavors. There are also "unnofficial" builds for IOS and Android.
+Cataclysm has official builds for Windows, Linux, and OSX, and each build comes in both terminal and graphical flavors. There are also "unofficial" builds for iOS and Android.
```

Comment:
I'm unsure about this change and I don't know what word to replace "platform" with. "platform" is wrong, because the platform (i.e. Windows) doesn't come in both flavors, the game does. "build" is also wrong, because the terminal build is a different build from the graphical build. Any ideas for a better word?

Reply:
Hrm, actually it's not just wording, it's no longer accurate.  Probably just need to enumerate the builds instead.
 Android AAB (ARM bundle)
Android APK (ARM 32 bit)
Android APK (ARM 64 bit)
Linux_x64 Curses
Linux_x64 Tiles
OSX Curses
OSX Tiles
Windows Tiles
Windows_x64 Tiles

Reply:
Could you do that? I don't really want to do major changes to the website myself because especially on this index page of the website, it's critical to give the impression of the game y'all want players to get. I've tried to only make small changes and retain things like sentence structure, "tone" of language, etc.